src/main/webapp/view/video/video.js

```diff
@@ -60,14 +62,14 @@ export default class Video {
         this.getCurrentVideoTimeMs.bind(this));
   }
 
-  /** Returns current video time of 'videoPlayer' in milliseconds. */
-  getCurrentVideoTimeMs() {
-    return secondsToMilliseconds(this.videoPlayer.getCurrentTime());
+  /** Seeks video to `timeMs`. */
+  seek(timeMs) {
+    this.#videoPlayer.seekTo(
+        millisecondsToSeconds(timeMs), /* allowSeekAhead= */ true);
   }
 
-  /** Seeks video to `currentTime`. */
-  seekVideo(timeMs) {
-    // TODO: Removed and implement.
-    console.log('SEEKING VIDEO TO: ' + timeMs);
+  /** Returns current video time of 'videoPlayer' in milliseconds. */
+  getCurrentVideoTimeMs() {
+    return secondsToMilliseconds(this.#videoPlayer.getCurrentTime());
   }
```

Comment:
I saw this code in a different PR, do you need to sync?

Reply:
Yes syncing now